intranet/static/js/common.nav.js

```diff
@@ -17,45 +14,56 @@ $(function() {
         }
     });
 
-    var initX = null, initY = null, listening = false;
-    var win = $(window);
-    win.on("touchstart", function(e) {
-        initX = e.originalEvent.touches[0].clientX;
-        initY = e.originalEvent.touches[0].clientY;
-        listening = true;
-    });
-    win.on("touchend", function() {
+    var initX = null,
+        initY = null,
         listening = false;
-    });
-    win.on("touchmove", function(e) {
-        if (!listening) {
-            return;
-        }
-        var nowX = e.originalEvent.touches[0].clientX;
-        var nowY = e.originalEvent.touches[0].clientY;
-        if (Math.abs(nowY - initY) > 30) {
+
+    $(window).on({
+        "touchstart": function(e) {
+            initX = e.originalEvent.touches[0].clientX;
+            initY = e.originalEvent.touches[0].clientY;
+            listening = true;
+        },
+        "touchend": function(e) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["e" is defined but never used](https://www.codacy.com/app/tjhsst-ion/ion/file/2769892631/issues/source?bid=3212528&fileBranchId=3355681#l27)